clarity/contracts/token/token-alex.clar

```diff
@@ -1,8 +1,6 @@
-(impl-trait .trait-sip-010.sip-010-trait)
-(impl-trait .trait-alex-token.dao-token-trait)
+(impl-trait .trait-pool-token.pool-token-trait)
```

Comment:
why alex and wbtc is pool token..? though there are no big difference aside from trait names?